### ISSUE TITLE
Can't Install in Google Collab

### ISSUE THREAD

**Issue by cedenoruel** [Shuyangzero/Ogre#1]
Hi Ogre Developers,
I am interested to use Ogre via Google Collab.

You can see my steps in the following link:

[Google Colab](https://colab.research.google.com/drive/13Vttf5sSqncHUMDBL9p2wJ9VYeEjUPI0?usp=sharing)

However, I get the following error message:
``
running install
running bdist_egg
running egg_info
writing ogre.egg-info/PKG-INFO
writing dependency_links to ogre.egg-info/dependency_links.txt
writing requirements to ogre.egg-info/requires.txt
writing top-level names to ogre.egg-info/top_level.txt
error: package directory 'ogre' does not exist
``

Any help would be highly appreciated. Thank you very much.

**Comment by Shuyangzero** [comment]
Hi @cedenoruel I already graduated so may not know the new context, if there's no respond from other collaborators you can send email to my Ph.D advisor Noa Marom for help. 
cc. @wwwaj Could you help to look at it?

**Comment by wwwaj** [comment]
Hi @cedenoruel,

Did you actually download the ogre package? I can see the ogre directory is indeed in the package. You should download the package and call the setup.py in the pacakge.

**Comment by cedenoruel** [comment]
> Hi @cedenoruel,
> 
> Did you actually download the ogre package? I can see the ogre directory is indeed in the package. You should download the package and call the setup.py in the pacakge.

Yes, I did. I downloaded and extracted the package contents using

`! git clone https://github.com/Shuyangzero/Ogre.git`

and I saw that the folder named "ogre" is created.

Have you ever tried installing the package in Colab Notebook (or is it even possible)? 

Thank you.
